.gitignore

```diff
@@ -35,8 +35,7 @@ cfgov-refresh/
 ._*
 .Spotlight-V100
 .Trashes
-Icon
-
+Icon


```

Comment:
Please revert this change. 

@chosak Any idea how to keep this in when most of us have a whitespace eliminator set up in our editor of choice?

Reply:
Sorry, I realize what's happening now, I had it in reverse in my mind.

@chosak I had the same thing happen when I edited the gitignore locally. The change from https://stackoverflow.com/questions/17556250/how-to-ignore-icon-in-git/21111738#21111738 seems to be converted to new lines when you save the gitignore in an editor. Any idea how we can prevent that?

Reply:
Unfortunately no; I see though that the way the change was made is pretty brittle and prone to being removed. Do we need this line? When are these files created -- only when browsing in an image folder on a Mac?

Reply:
It seems that way. I have never seen one included before but I also think it's such a standard that every repo I've worked in since starting with git has included the old pattern you had updated. I guess it's worth removing and keeping an eye out for unintended icon files?